include/config/config_rom.h

```diff
@@ -28,3 +28,6 @@
  */
 #define BORDER_HEIGHT_CONSOLE  0
 #define BORDER_HEIGHT_EMULATOR 0
+
+// Informs supported emulators to default to gamecube controller inputs
```

Comment:
Ah crap I didn't see this initially, can somebody change this to be a block comment?